spec/filters/alter_spec.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-require "logstash/devutils/rspec/spec_helper"
+require_relative '../spec_helper'
```

Comment:
as said, just curious, most of the plugins in logstash-plugins were done prior to the plugin generator. maybe @ph have a stronger point of view here ?